src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/IProtocolMangler.kt

```diff
@@ -64,7 +64,7 @@ interface IProtocolMangler : AutoCloseable {
      * @param message buffer with sent message
      * @param metadata message metadata
      */
-    fun afterOutgoing(message: ByteBuf, metadata: Map<String, String>) = Unit
+    fun afterOutgoing(message: ByteBuf, metadata: MutableMap<String, String>) = Unit
```

Comment:
```suggestion
    fun afterOutgoing(message: ByteBuf, metadata: Map<String, String>) = Unit
```
This method is called when a message is already sent. So there's no point in modifying anything in it